sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTestBase.java

```diff
@@ -620,26 +665,52 @@ void healthcareLroRunner(BiConsumer<List<TextDocumentInput>, RecognizeHealthcare
             asList(
                 new TextDocumentInput("0", HEALTHCARE_INPUTS.get(0)),
                 new TextDocumentInput("1", HEALTHCARE_INPUTS.get(1))),
-            new RecognizeHealthcareEntityOptions().setIncludeStatistics(true));
+            new RecognizeHealthcareEntityOptions().setIncludeStatistics(true).setPollInterval(durationTestMode));
     }
 
     void healthcareLroPaginationRunner(
-        BiConsumer<List<TextDocumentInput>, RecognizeHealthcareEntityOptions> testRunner) {
+        BiConsumer<List<TextDocumentInput>, RecognizeHealthcareEntityOptions> testRunner, int totalDocuments) {
         List<TextDocumentInput> documents = new ArrayList<>();
-        // Service has 20 as the default size per page. So there will be 2 remaining page in the next page link
-        for (int i = 0; i < 22; i++) {
+        // Service has 10 as the default size per page. So there will be 2 remaining page in the next page link
```

Comment:
Do we have this mentioned in the docs anywhere?

Reply:
Unfortunately. This is the first - preview of Healthcare and Analyze Tasks. There is no written document right now. Service  will increase the max document size in the next release